frontend/package.json

```diff
@@ -35,6 +35,7 @@
     "lint:scss:fix": "yarn run prettier -c resources/styles -c src/**/*.scss -w",
     "build:test": "clojure -M:dev:shadow-cljs compile test",
     "test": "yarn run build:test && node target/tests/test.js",
+    "test-storybook": "test-storybook",
```

Comment:
This should follow the same style of commands

Reply:
Changed to `test:storybook`